src/main/java/uk/gov/digital/ho/hocs/audit/aws/config/LocalStackConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.digital.ho.hocs.audit.aws.config;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.sqs.AmazonSQSAsync;
+import com.amazonaws.services.sqs.AmazonSQSAsyncClientBuilder;
+import com.amazonaws.services.sqs.model.PurgeQueueRequest;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.aws.messaging.config.SimpleMessageListenerContainerFactory;
+import org.springframework.cloud.aws.messaging.config.annotation.EnableSqs;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+
+@EnableSqs
+@Configuration
+@Profile({"local"})
+public class LocalStackConfiguration {
+
+    @Primary
+    @Bean
+    public AmazonSQSAsync awsSqsClient(
+            @Value("${aws.queue.audit.name}") String queueName,
+            @Value("${aws.sqs.config.url}") String awsBaseUrl) {
+        BasicAWSCredentials credentials = new BasicAWSCredentials("test", "test");
```

Comment:
nitpick but I'd just inline lines 27 and 29 in the builder, they were probably only separate for easy breakpointing